src/control/lib/support/log.go

```diff
@@ -435,8 +437,40 @@ func getSysNameFromQuery(configPath string, log logging.Logger) ([]string, error
 	return hostNames, nil
 }
 
+func customCopy(log logging.Logger, opts CollectLogsParams, fileTransferExec string) error {
+	cmd := strings.Join([]string{
+		fileTransferExec,
+		opts.TargetFolder,
+		opts.FileTransferExecArgs},
+		" ")
+
+	out, err := exec.Command("sh", "-c", cmd).Output()
+	if err != nil {
+		return errors.Wrapf(err, "Error running command %s %s", cmd, string(out))
+	}
+	log.Infof("customCopy:= %s stdout:\n%s\n\n", cmd, string(out))
+	return nil
+}
+
 // R sync logs from individual servers to Admin node
 func rsyncLog(log logging.Logger, opts ...CollectLogsParams) error {
+	var cfgPath string
+	if opts[0].Config != "" {
+		cfgPath = opts[0].Config
+	} else {
+		cfgPath, _ = getServerConf(log)
```

Comment:
Not sure the context, but I would strongly suggest that we don't ignore errors in general.

Though the error flow is kind of weird in that function as well. Seems like it tries to return something even when the error is set? Not sure what that's about.